doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -92,6 +92,7 @@ Matter
     8 dBm for nRF52840, 3 dBm for nRF5340, 20 dBm for all devices with FEM enabled, and 0 dBm for sleepy devices.
 
 * Fixed the issue of connection timing out when attaching to a Wi-Fi access point that requires Wi-Fi Protected Access 3 (WPA3).
+* Fixed ``NetworkInterfaces`` attribute of General Diagnostics cluster to return MAC Extended Address instead of EUI-64 for Thread network interfaces.
```

Comment:
```suggestion
* Fixed:

   * The issue where the connection would time out when attaching to a Wi-Fi access point that requires Wi-Fi Protected Access 3 (WPA3).
   * The issue where the ``NetworkInterfaces`` attribute of General Diagnostics cluster would return MAC Extended Address instead of EUI-64 for Thread network interfaces.
```

Reply:
The other way around (EUI-64 instead of MAC Extended Address), but I applied the change.